src/data/graph-data.js

```diff
@@ -79,6 +81,49 @@ async function getLinkedIssues(
   });
 }
 
+export async function GetAllOrganizations(
+  endpointUrl,
+  zenhubApiKey,
+  signal
+) {
+  const gqlQuery = createGqlQuery(endpointUrl, zenhubApiKey, signal);
+
+  const {
+    viewer: {
+      zenhubOrganizations: {
+        nodes: organizations,
+      }
+    }
+  } = await gqlQuery(GET_ALL_ORGANIZATIONS, "GetAllOrganizations", {});
+
+  return organizations.map( ( organization ) => ( {
+    id: organization.id,
+    name: organization.name,
+    workspaces: organization.workspaces.nodes.map( ( workspace ) => ( { id: workspace.id, name: workspace.name } ) ),
```

Comment:
This can be simplified a touch:

```suggestion
    workspaces: organization.workspaces.nodes.map( ( {id, name} ) => ( { id, name } ) ),
```

Reply:
Updated, thank you!